payroll/gql_queries.py

```diff
@@ -40,6 +40,8 @@ class Meta:
         filter_fields = {
             "id": ["exact"],
             "name": ["iexact", "istartswith", "icontains"],
+            "status": ["exact", "iexact", "startswith", "istartswith", "contains", "icontains"],
+            "payment_method": ["exact", "iexact", "startswith", "istartswith", "contains", "icontains"],
```

Comment:
Do we need both case sensitive and insensitive filters?

Reply:
I think if we have ENUM statuses and methods - we could stay with having only case sensitive ones